src/js/lib/monster.ui.js

```diff
@@ -475,9 +475,34 @@ define(function(require) {
 				options = _.merge({
 					search_contains: true,
 					width: '220px'
-				}, pOptions);
+				}, pOptions),
+				chosen;
 
 			$target.chosen(options);
+
+			if (!pOptions.tags) {
+				return;
+			}
+
+			chosen = $target.data('chosen');
+
+			// Bind the keyup event to the search box input
+			chosen.container.find('input').on('keyup', function(e) {
+				// If we hit Enter and the results list is empty (no matches) add the option
+				if (e.which !== 13 || chosen.dropdown.find('li.no-results').length === 0) {
+					return;
+				}
+
+				var newTag = $(this).val(),
+					option = $('<option>').val(newTag).text(newTag);
+
+				// Add the new option
+				$target.prepend(option);
+				// Automatically select it
+				$target.find(option).prop('selected', true);
```

Comment:
A more straightforward way of doing this it to set the `select` value directly:
```suggestion
				$target.val(newTag);
```

Reply:
Looking trough the code this line isn't required, I just add the `prop('selected', true)` on the `option` generation

https://github.com/2600hz/monster-ui/pull/392/files#diff-602a66ca60a9e72a9cdcf92ae9ec90cdR500